src/recipes/heartbeat-by-user-first-impression/config.js

```diff
@@ -12,15 +12,71 @@
 
 "use strict";
 
-let million = Math.pow(10,6);
-let thousand = Math.pow(10,3);
-let percent = 0.01;
-let days = 24 * 60 * 60 * 1000;
+const VERSION=33;
 
-/*
-  days since last ask?  -- function or constant?
+/** EngagementUrl Rules
 
-*/
+  Notes:
+  1.  first match will win.  PUT MORE SPECIFIC RULES FIRST.
+      Note: NO automated check of rule order.
+  2.  'qsurvey' urls will postpend all the right query args.  others won't.
+      (Write a bug if this is a problem!)
+  3.  If you need more fields, dig into `src/common/personinfo.js` .
+  4.  `npm run surveys` will attempt to run some common configs through
+      the rules.
+  5.  `test/recipe/test-heartbeat-by-user-first-impression.js` does some
+      additional linting.
+ */
+
+const engagementRules = [
+  // en, aurora.
+  {
+    alias: '^en, ^aurora',
+    rule: {
+      locale: /^en/,
+      channel: /^aurora/
+    },
+    urls: [
```

Comment:
This is supported.  I still don't like the unified in objects, but at line 42, 'asBreaks' is shown, which does that rescale :)